test/db/rzil/hexagon

```diff
@@ -0,0 +1,389 @@
+# These tests run the test binaries from QEMU.
+# Only bineries which contain 100% supported instructions are run here.
+# If one of those tests fails and you need to debug it,
+# please use BAP qemu to generate a trace of it
+# and compare it with rz-tracetest to Rizin.
+# The broken instructions will mismatch.
+
+NAME=Run test_vspliceb
+FILE=bins/elf/hexagon/rzil/test_vspliceb
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vpmpyh
+FILE=bins/elf/hexagon/rzil/test_vpmpyh
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vminh
+FILE=bins/elf/hexagon/rzil/test_vminh
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vmaxh
+FILE=bins/elf/hexagon/rzil/test_vmaxh
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vlsrw
+FILE=bins/elf/hexagon/rzil/test_vlsrw
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vcmpw
+FILE=bins/elf/hexagon/rzil/test_vcmpw
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vcmpb
+FILE=bins/elf/hexagon/rzil/test_vcmpb
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_vavgw
+FILE=bins/elf/hexagon/rzil/test_vavgw
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_round
+FILE=bins/elf/hexagon/rzil/test_round
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_reorder
+FILE=bins/elf/hexagon/rzil/test_reorder
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_packet
+FILE=bins/elf/hexagon/rzil/test_packet
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_mpyi
+FILE=bins/elf/hexagon/rzil/test_mpyi
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_lsr
+FILE=bins/elf/hexagon/rzil/test_lsr
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_jmp
+FILE=bins/elf/hexagon/rzil/test_jmp
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_hwloops
+FILE=bins/elf/hexagon/rzil/test_hwloops
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_hl
+FILE=bins/elf/hexagon/rzil/test_hl
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_fibonacci
+FILE=bins/elf/hexagon/rzil/test_fibonacci
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_ext
+FILE=bins/elf/hexagon/rzil/test_ext
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_dotnew
+FILE=bins/elf/hexagon/rzil/test_dotnew
+TIMEOUT=5
+CMDS=<<EOF
+e io.cache=1
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
```

Comment:
should we expect some output somewhere? or maybe a register dump after the execution?

Reply:
Indeed. A register dump would be good. Let me add this later.

Reply:
On a second thought, it might not be too useful. The register dump would never be printed in case of a timeout. And if everything executed correctly, it says pretty much nothing interesting anyways.

Would you agree?